system/lib/cmake/SDL3/sdl3-config.cmake

```diff
@@ -0,0 +1,18 @@
+if(NOT TARGET SDL3::SDL3)
```

Comment:
The SDL2 version of this file sets SDL2_INCLUDE_DIRS and SDL2_LIBRARIES.. are those not needed in this case?

Reply:
Nope, since SDL3 is now target based and no longer uses the old style variables, so linking to one of the `SDL3::*` targets is enough to set it up correctly. See https://wiki.libsdl.org/SDL3/README/cmake where there's no mention of the old style `<name>_LIBRARIES`/`<name>_INCLUDE_DIRS`.